Resources/Textures/_Harmony/Mobs/Pets/gizard.rsi/meta.json

```
@@ -0,0 +1,21 @@
+﻿
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Lizard Sprite Made by Mithrandalf, Hat made by Flarbergaster",
+  "states": [
+    {
+      "name": "gizard",
+      "directions": 4
+    },
+    {
+      "name": "gizarddead",
+      "directions": 1
```

The state only having 1 direction is already assumed if you don't define it
```suggestion
      "name": "gizarddead"
```